libvast/vast/format/reader.hpp

```diff
@@ -13,16 +13,17 @@
 
 #pragma once
 
-#include <cstddef>
-#include <string_view>
+#include "vast/defaults.hpp"
+#include "vast/fwd.hpp"
+#include "vast/system/accountant.hpp"
```

Comment:
Why does the `reader` interface need to know about the `accountant`?

Reply:
Essentially because that's where `system::report` lives, but I can just as easily move this into a separate header.